frontend/src/components/pages/ForgotPassword.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { useState } from "react";
+import axios from "axios";
+
+
+const ForgotPassword = () => {
+  const [email, setEmail] = useState("");
+  const [error, setError] = useState("");
+  const [success, setSuccess] = useState("");
+
+  const forgotPasswordHandler = async (e) => {
+    e.preventDefault();
+
+    const config = {
+      header: {
+        "Content-Type": "application/json",
+      },
+    };
+
+    try {
+      const { data } = await axios.post(
+        "http://localhost:8080/api/auth/forgotpassword",
+        { email },
```

Comment:
Use a .env var for backend_base_url. We could aslo have separate .env files for frontend and back end. 

This comment is the same for the rest of the /api/auth/ routes.